app/src/sharedTest/java/org/oppia/app/topic/review/TopicReviewFragmentTest.kt

```diff
@@ -123,6 +135,58 @@ class TopicReviewFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicReviewFragment_loadFragment_configurationChange_checkTopicThumbnail_isCorrect() {
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      onView(
+        allOf(
+          withText(TopicTab.getTabForPosition(3).name),
+          isDescendantOfA(withId(R.id.topic_tabs_container))
+        )
+      ).perform(click())
+      it.onActivity { activity ->
+        activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
+      }
+      it.recreate()
+      onView(withId(R.id.review_recycler_view)).check(matches(hasDescendant(withDrawable(subtopicThumbnail))))
+    }
+  }
+
+  @Test
+  fun testTopicReviewFragment_loadFragment_configurationChange_selectReviewTopics_reviewCardDisplaysCorrectExplanation() {
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      onView(
+        allOf(
+          withText(TopicTab.getTabForPosition(3).name),
+          isDescendantOfA(withId(R.id.topic_tabs_container))
+        )
+      ).perform(click())
+      it.onActivity { activity ->
+        activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
+      }
+      it.recreate()
+      onView(atPosition(R.id.review_recycler_view, 0)).perform(click())
+      onView(withId(R.id.review_card_explanation_text)).check(matches(withText("Description of subtopic is here.")))
```

Comment:
Do not check this. Instead check if intent has been passed correctly or not because `TopicReview` is not responsible for setting data inside reviewcard.

Reply:
Changes done